UnitTests/UnitTests.HighPerformance.Shared/Extensions/Test_ArrayExtensions.2D.cs

```diff
@@ -233,39 +233,209 @@ public void Test_ArrayExtensions_2D_GetRow_Empty()
         {
             int[,] array = new int[0, 0];
 
+            // Try to get a row from an empty array (the row index isn't in range)
             Assert.ThrowsException<ArgumentOutOfRangeException>(() => array.GetRow(0).ToArray());
         }
 
         [TestCategory("ArrayExtensions")]
         [TestMethod]
         [SuppressMessage("StyleCop.CSharp.NamingRules", "SA1312", Justification = "Dummy loop variable")]
         [SuppressMessage("StyleCop.CSharp.LayoutRules", "SA1501", Justification = "Empty test loop")]
```

Comment:
There a few `SuppressMessageAttributes` left on methods that are no longer needed. It would be good to remove them from methods that no longer need it.

Reply:
Oh, good catch, forgot about those! 😄
Fixed in 7e67a9c84bb89a1500fd139739ddf8a91fecece2, and also fixed some other leftover warnings in the tests.